src/theme/SearchBar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import EnhancedSearch from 'enhancedocs-search';
+
+import 'enhancedocs-search/dist/style.css';
+
+export default function SearchBarWrapper(props) {
+  return (
+    <EnhancedSearch
+      config={{
+        enhancedSearch: {
+          projectId: "<replace_with_project_id>",
+          accessToken: "<replace_with_access_token>"
```

Comment:
What is the access token?

Reply:
Is the publisable key from https://app.enhancedocs.com/settings/api-keys. but good to mention, that's no very clear...